src/common/meta/src/key/table_route.rs

```diff
@@ -62,29 +62,57 @@ impl TableRouteValue {
     }
 
     /// Returns a new version [TableRouteValue] with `region_routes`.
-    pub fn update(&self, region_routes: Vec<RegionRoute>) -> Self {
+    ///
+    /// # Panics
+    /// The route type is not the [TableRouteValue::Physical].
```

Comment:
Will not panic now:
```suggestion
    /// Returns [UnexpectedTableRouteType] if the route type is not the [TableRouteValue::Physical].
```
same for other places below